spacy/language.py

```diff
@@ -1719,7 +1721,12 @@ def from_config(
                                     name, source_name, pipe_cfg["replace_listeners"]
                                 )
                                 listeners_replaced = True
-                    nlp.add_pipe(source_name, source=source_nlps[model], name=pipe_name)
+                    with warnings.catch_warnings():
+                        warnings.filterwarnings("ignore", message="\\[W113\\]")
+                        nlp.add_pipe(source_name, source=source_nlps[model], name=pipe_name)
+                    if model not in source_nlp_vectors_hashes:
+                        source_nlp_vectors_hashes[model] = hash(source_nlps[model].vocab.vectors.to_bytes())
```

Comment:
Ah this will take forever :(. We'd at least want to use murmurhash or something?

If we absolutely need to make some sort of signature, can we use a subset? Like maybe, first column, last column, first row, last row, size?

I just really don't think we can have this sort of overhead for a verification check. I could maybe consider it for an option that's off by default, but definitely not one that's on by default.

Reply:
Hm actually this only runs on sourcing right, not on every load?

I'm still not really thrilled about it, but it's less bad than I first thought.

Reply:
In the situation where the alternative is that models fail silently, I think a few seconds for this check is okay. It should only run once for each specified source model, so it most use cases only for 1-2 models total. In tests with `spacy assemble`, the additional time wasn't particularly noticeable given that it also takes a few seconds to load models with vectors.